cmd/aida-vm-sdb/run_substate_test.go

```diff
@@ -214,12 +220,18 @@ func TestVmSdb_Substate_ValidationDoesNotFailOnValidTransaction(t *testing.T) {
 	ctrl := gomock.NewController(t)
 	provider := executor.NewMockProvider[txcontext.TxContext](ctrl)
 	db := state.NewMockStateDB(ctrl)
+
+	chainConfig, err := utils.GetChainConfig(utils.EthTestsChainID)
```

Comment:
Is this a typo? Did you mean `utils.MainnetChainID`? This is `aida-vm-sdb substate` test.